libc/src/stdfix/CMakeLists.txt

```diff
@@ -2,6 +2,20 @@ if(NOT LIBC_COMPILER_HAS_FIXED_POINT)
   return()
 endif()
 
+foreach(suffix IN ITEMS hr r lr hk k lk uhr ur ulr uhk uk ulk)
```

Comment:
nit: you can let `countls` targets to share the loop with `round`

Reply:
This is fixed in 97377a4.

https://github.com/llvm/llvm-project/blob/97377a4ee6a92cd1a46c1d57d7c7ee75d18f8790/libc/test/src/stdfix/CMakeLists.txt#L45-L75

Reply:
This is fixed in cc87a9a.

https://github.com/llvm/llvm-project/blob/cc87a9ac973131db1047a22ebfb2b0db2d2f1926/libc/src/stdfix/CMakeLists.txt#L33-L57